detections.py

```diff
@@ -77,7 +77,7 @@ def plot_one_box_tlwh(x, img, color=None, label=None, line_thickness=None):
         cv2.putText(img, label, (c1[0], c1[1] - 2), 0, tl / 3, [225, 255, 255], thickness=tf, lineType=cv2.LINE_AA)
 
 
-def yolo_predict(yolov5, img, im0s):
+def yolo_predict(yolov5, img, im0s,log):
```

Comment:
add a logger as a parameter so that the TrafficTracker can pass it's logger to it at inference time